sdk/servicenetworking/Azure.ResourceManager.ServiceNetworking/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
Change the `Unreleased` to the expect date when need us to release the lib to nuget (2 weeks after pr published at least)